rust/toolchain.bzl

```diff
@@ -179,20 +180,20 @@ def _make_libstd_and_allocator_ccinfo(ctx, rust_std, allocator_library):
 
         # The libraries panic_abort and panic_unwind are alternatives.
         # The std by default requires panic_unwind.
-        # Exclude panic_abort if panic_unwind is present.
-        # TODO: Provide a setting to choose between panic_abort and panic_unwind.
+        # Exclude panic_abort if panic_unwind is present, and vice versa.
         filtered_between_core_and_std_files = rust_stdlib_info.between_core_and_std_files
-        has_panic_unwind = [
+        if panic == "abort":
+            panic_filter = "panic_unwind"
```

Comment:
I believe there is also `libunwind` that we want to filter out in `--panic=abort` mode?

Reply:
I think that's used for calculating stacktraces independent of the panic mode. Also I think I found the part of rustc that implements the switching logic at https://github.com/rust-lang/rust/blob/42faef503f3e765120ca0ef06991337668eafc32/compiler/rustc_metadata/src/creader.rs#L746, and it's only swapping the `panic_unwind` and `panic_abort` crates, not doing anything with `libunwind`.